src/utils/explorers.js

```diff
@@ -0,0 +1,23 @@
+import { SUPPORTED_NETWORKS, isMainNet } from '../utils/networkUtils';
+
+const getEtherScanBaseURL = networkId => {
+	const network = SUPPORTED_NETWORKS[networkId];
+
+	if (isMainNet(networkId) || network == null) {
+		return 'https://etherscan.io';
```

Comment:
@clementbalestrat I wasn't sure what do we want to return when the network is invalid, so I just default it to MainNet. Let me know if you want this changed.

Reply:
Yeah that sounds good for now!